test/test_kiwipiepy.py

```
@@ -175,6 +175,49 @@ def test_pretokenized():
+    kiwi = Kiwi()
+
+    assert [
+        (token.form, token.start, token.end) for token in kiwi.tokenize('😀 밥을 먹었다')
+    ] == [('😀', 0, 1), ('밥', 2, 3), ('을', 3, 4), ('먹', 5, 6), ('었', 6, 7), ('다', 7, 8)]
+
+    assert [
+        (token.form, token.start, token.end) for token in kiwi.tokenize('밥😀😀 먹었다')
+    ] == [('밥', 0, 1), ('😀', 1, 2), ('😀', 2, 3), ('먹', 4, 5), ('었', 5, 6), ('다', 6, 7)]
+
+    texts = ['오늘 밥을 먹었다', '𠀀 밥을 먹었다', '🇰🇷 밥을 먹었다']
+    for text, tokens in zip(texts, kiwi.tokenize(iter(texts))):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)